requirements.txt

```diff
@@ -1,2 +1,2 @@
 click-option-group>=0.5.5
-cirrus-geo>=0.9.0a0
+cirrus-geo>=0.9.0
```

Comment:
I suppose this could stay at the `0.9.0a0` release, but somehow it feels good to bump it up to the official `0.9.0`.